test/e2e/storage/mounted_volume_resize.go

```diff
@@ -43,7 +43,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 )
 
-var _ = utils.SIGDescribe("Mounted volume expand", func() {
+var _ = utils.SIGDescribe("Mounted volume expand [Feature:StorageProviderGCEPD]", func() {
```

Comment:
@gnufied do we still need these tests or do the testsuite tests cover it? https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/volume_expand.go

Reply:
volume expand tests are also covered in pdcsi migration tests

Reply:
Yes, what I mean is are these tests in mounted_volume_resize still needed or do we have equivalent test cases in testsuites/volume_expand.go

Reply:
I think this test in mounted_volume_expand will also run due to focus `[V|v]olume\sexpand`